doc/dependency_decisions.yml

```diff
@@ -243,3 +243,9 @@
     :why: License approved by Red Hat
     :versions: []
     :when: 2023-04-14 08:25:18.115879534 Z
+- - :approve
+  - nio4r
+  - :who:
+    :why:
+    :versions: []
+    :when: 2024-03-08 23:15:47.934892299 Z
```

Comment:
I don't remember, but I think `:approve` is for packages, and `:permit` is for licenses :thinking: 

I was going to add "I don't remember approving it manually, probably was done by the tool"... but the explicit commit messages says otherwise :upside_down_face: 

I'll figure this out.

Reply:
OK, so the result of `bundle exec rake licenses:compliance`:

```
Dependencies that need approval:
nio4r, 2.7.0, "MIT AND (BSD-2-Clause OR GPL-2.0-or-later)"
```

Reply:
But the good news is that recently a new version was released, that has the licences info improved, and now we don't need this exception.
I've updated `Gemfile.lock`